dask/dataframe/tests/test_dataframe.py

```diff
@@ -470,9 +470,6 @@ def test_describe_empty():
         df_none.describe(), ddf_none.describe(percentiles_method="dask").compute()
     )
 
-    with pytest.raises(ValueError):
-        ddf_len0.describe(percentiles_method="dask").compute()
-
     with pytest.raises(ValueError):
         ddf_len0.describe(percentiles_method="dask").compute()
```

Comment:
This test appears to be run twice (unless I'm missing something). So have just dropped one of these